viewer/packages/api/src/api-helpers/Image360ApiHelper.ts

```diff
@@ -294,7 +294,7 @@ export class Image360ApiHelper {
   }
 
   public exit360Image(): void {
-    this._image360Facade.allIconsVisibility = true;
+    this._interactionState.currentImage360Entered?.icon.setVisibility(true);
```

Comment:
If the 360 image collection is hidden when this happens, the icon will be visible while the rest are hidden.

Reply:
Fixed: See https://github.com/cognitedata/reveal/pull/3074/commits/951c27062f3171e6b7aa67dbc863d857165bde6f

Reply:
Nice!